pom.xml

```diff
@@ -0,0 +1,341 @@
+                            </suiteXmlFiles>
+                            <excludes>
+                                <exclude>com.browserstack.test.suites.offers.OfferTest</exclude>
+                            </excludes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>bstack-parallel</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <environment>remote</environment>
+                                <test>parallel</test>
+                            </systemPropertyVariables>
+                            <properties>
+                                <property>
+                                    <name>listener</name>
+                                    <value>com.browserstack.test.utils.BrowserstackTestStatusListener</value>
+                                </property>
+                            </properties>
+                            <parallel>${testng.parallel}</parallel>
+                            <threadCount>${testng.threadCount}</threadCount>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>bstack-parallel-browsers</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <environment>remote</environment>
+                                <test>parallel</test>
+                            </systemPropertyVariables>
+                            <properties>
+                                <property>
+                                    <name>listener</name>
+                                    <value>com.browserstack.test.utils.BrowserstackTestStatusListener</value>
+                                </property>
+                            </properties>
+                            <parallel>${testng.parallel}</parallel>
+                            <threadCount>${testng.threadCount}</threadCount>
+                            <suiteXmlFiles>
+                                <suiteXmlFile>testng-browserstack-parallel.xml</suiteXmlFile>
+                            </suiteXmlFiles>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
Only Launching Chrome and FireFox  need to launch test for Safari/IE and Mobile browsers both IOS and Android.